data/vocab/lite.ttl

```diff
@@ -321,6 +321,33 @@ bld:rightsStatement a rdf:Property ;
     rdfs:range rdfs:Literal ;
     rdfs:domain bld:Resource .
 
+bld:type a rdf:Property ;
+    rdfs:label "type"@en ;
+    rdfs:comment "Text string representing the type of resource."@en ;
+    owl:equivalentProperty dc:type ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bld:Resource .
+
+bld:createdDate a rdf:Property ;
+    rdfs:label "created date"@en ;
+    rdfs:comment "The date a Resource was created."@en ;
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
+
+bld:updatedDate a rdf:Property ;
+    rdfs:label "updated date"@en ;
+    rdfs:comment "The date a Resource was last updated."@en ;
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
```

Comment:
Check to see if subProperty inherents the range from date. If it does remove the range reference here